certdocs/0-DDLN-100-ORD.md

```diff
@@ -39,9 +39,9 @@ This document discusses the following topics:
 
 #### Internal Documents
```

Comment:
At the moment `ReqTypeToDocType ` in the configuration file (currently https://github.com/daedaleanai/reqtraq/blob/master/config/config_ddln.go) can be changed to specify in which files the requirements are defined. 

If "ORD/SRD/SDD" must appear in the final reports, the change will benefit us as well.